prow/overlays/cnv-prod/labels_operate-first.yaml

```diff
@@ -76,6 +100,42 @@ default:
       target: both
       prowPlugin: label
       addedBy: anyone
+    - color: c7def8
+      description: Indicates an issue/PR is related to governance.
+      name: kind/governance
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: c7def8
+      description: Indicates an issue/PR is related to a handbook.
+      name: kind/handbook
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: c7def8
+      description: Indicates an issue/PR is related to marketing.
+      name: kind/marketing
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: c7def8
+      description: Indicates an issue/PR is related to metrics.
+      name: kind/metrics
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: c7def8
+      description: Indicates an issue/PR is related to project news and outreach.
+      name: kind/news
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: c7def8
+      description: Indicates an issue/PR is related to onboarding.
+      name: kind/onboarding
```

Comment:
Should this one replace [`onboarding`](https://github.com/operate-first/support/labels/onboarding)?

What should we do with [`onboarding-argocd`](https://github.com/operate-first/support/labels/onboarding-argocd)?

Reply:
That is an interesting nuance. I am thinking about the steps of onboarding individuals into the project, but that includes in some cases onboarding their project code into the infrastructure.

If I get the philosophy correct, should it be:

`kind/onboarding` and `area/cluster`|`area/argocd`

I'm not trying to create a proliferation of labels, my intention here was to try getting most of the labels I think I'll need for labeling issues/PRs across the project for their relation to aspects of the community.

Reply:
Yeah, I think the `kind/onboarding` + either `area/cluster` or `area/argocd` would be a good option.

> I'm not trying to create a proliferation of labels, my intention here was to try getting most of the labels I think I'll need for labeling issues/PRs across the project for their relation to aspects of the community.

I'm with you. If we can bring some order to it, it'll be nice :slightly_smiling_face: